### PR TITLE
Add command line arg to use the leapp beta repos

### PR DESCRIPTION
**Pull request by toddr**

### CODE REVIEW

File: lib/Elevate/OS/CentOS7.pm [hunk 1]
```diff
@@ -24,6 +24,8 @@ use constant available_upgrade_paths => (
 use constant default_upgrade_to => 'AlmaLinux';
 use constant ea_alias           => 'CentOS_8';
 use constant elevate_rpm_url    => 'https://repo.almalinux.org/elevate/elevate-release-latest-el7.noarch.rpm';
+use constant leapp_repo_prod    => 'elevate';
+use constant leapp_repo_beta    => '';
```
Comment (toddr) [inline lib/Elevate/OS/CentOS7.pm:28]: :( almalinux doesn't provide their beta repos in their repo file. Maybe we can get them to add it!

**Comment by toddr** [comment]
Testing: check whether the repo is enabled by doing: 
```
egrep '^\[|^enabled' /etc/yum.repos.d/cloudlinux-elevate.repo
```

File: lib/Elevate/OS/CentOS7.pm (hunk 1 above)
Comment (cPholloway) [reply lib/Elevate/OS/CentOS7.pm:28]: CONSIDER:  Move line 28 to `Elevate::OS::RHEL` as a fallback instead of defining it here

**Review by cPholloway — CHANGES_REQUESTED** [review]

File: script/elevate-cpanel.PL [hunk 2]
```diff
@@ -415,8 +416,10 @@ sub run ( $pkg, @args ) {
     }
 
     if ( $self->getopt('start') ) {
-        die qq[Unsupported option with --start\n] if $self->getopt('continue') || $self->getopt('service');
-        return 1                                  if $self->start();
+        die qq[Unsupported option with --start\n]        if $self->getopt('continue') || $self->getopt('service');
+        die qq[BETA is unavailable for this leapp repo.] if $self->getopt('leappbeta') && !length Elevata::OS::leapp_repo_beta();
```
Comment (annes449) [inline script/elevate-cpanel.PL:420]: Elevata::OS s/b Elevate::OS

File: lib/Elevate/OS/CentOS7.pm (hunk 1 above)
Comment (toddr) [reply lib/Elevate/OS/CentOS7.pm]: Good idea! done.

File: script/elevate-cpanel.PL (hunk 2 above)
Comment (toddr) [reply script/elevate-cpanel.PL]: oof! fixed.

**Review by cPholloway — APPROVED** [review]

File: lib/Elevate/Usage.pm [hunk 3]
```diff
@@ -23,6 +23,7 @@ sub _OPTIONS {
       upgrade-to=s
       no-leapp
       non-interactive
+      leappbeta
```
Comment (toddr) [inline lib/Elevate/Usage.pm:26]: @cPholloway this is all I need to do?

Comment (cPholloway) [reply lib/Elevate/Usage.pm:26]: Should this option only be used with `--start`?  If so, can you add it to the `start_only_options` array as well?

**Review by cPholloway — APPROVED** [review]

**Review by annes449 — APPROVED** [review]
All good with QA

**Pull request merged**